core/readpref/options.go

```diff
@@ -34,14 +34,15 @@ func WithMaxStaleness(ms time.Duration) Option {
 // overrides all previous calls to either method.
 func WithTags(tags ...string) Option {
 	return func(rp *ReadPref) error {
-		if len(tags)%2 != 0 {
+		l := len(tags)
```

Comment:
Can we change `l` here to `length`? It's a little confusing reading the code because `l` looks a lot like `1`, and line 42 was confusing for a moment because it looks like `1/2` instead of `l/2`.

Reply:
Done.